devices/AtModem/Http/Http/HttpVersion.cs

```diff
@@ -0,0 +1,29 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+//
+
+namespace System.Net
+{
+
+    /// <summary>
+    /// Defines the HTTP version numbers that are supported by the
+    /// <see cref="HttpWebRequest"/> and
```

Comment:
Can't corss ref to `HttpWebRequest` nor `HttpWebResponse`

Reply:
cleared. Good cat h!